### PR TITLE
(maint) Symlinks need to point to files that exist

### PR DESCRIPTION
**Pull request by melissa**
Prior to this commit, all debian/ubuntu cows require a script to build.
This script was a single copy for an older debian/ubuntu build, but it
was recently switched to a more generic name, either 'debian' or
'ubuntu', so when the older platforms reach EOL, removing them will be a
lot simpler. This is great, unless the 'debian' and 'ubuntu' symlink
sources don't actually exist

### CODE REVIEW

File: manifests/setup/cows.pp [hunk 1]
```diff
@@ -122,7 +118,7 @@
   # cows of. All Debian varieties are symlinks now pointing to `debian` and all Ubuntu
   # varieties are symlinks pointing to `ubuntu`.
   # We also lay down the script for cumuluslinux which is currently a copy of wheezy.
-  debbuilder::util::file_on_disk { ['CumulusLinux-2.2', 'jessie', 'lucid', 'precise', 'sid', 'squeeze', 'stable', 'testing', 'trusty', 'unstable', 'utopic', 'vivid', 'wheezy', 'wily' ]:
+  debbuilder::util::file_on_disk { ['debian', 'ubuntu', $cows]:
```
Comment (melissa) [inline manifests/setup/cows.pp]: I have no idea if this will work.

cc/ @underscorgan ?

**Comment by stahnma** [comment]
LGTM

File: manifests/setup/cows.pp (hunk 1 above)
Comment (underscorgan) [inline manifests/setup/cows.pp]: :+1: yep, this should work!

**Comment by underscorgan** [comment]
:+1:

**Pull request merged**